prow/config.yaml

```diff
@@ -87,23 +67,35 @@ branch-protection:
       required_pull_request_reviews:
         required_approving_review_count: 1
       restrictions:
-        teams:
-        - repo-admins
       repos:
         api:
           protect: false
           branches:
             <<: *blocked_branches
-            release-1.2:
+            master:
               protect: true
-              required_status_checks:
-                contexts:
-                - "merges-blocked-needs-admin"
-            release-1.1:
+        istio.io:
+          protect: false
```

Comment:
What branches do we want to leave unprotected and why?

Reply:
The non-official branches are left unprotected. So branches developers create in an ad-hoc fashion as part of doing a PR. We discourage the practice, but it's common nevertheless.